files/image_config/get_graph/get_graph

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+while [ ! -f /etc/sonic/graph_service_url ]; do
+    echo "No graph service url speficied, waiting for DHCP response..."
+    sleep 1
+done
```

Comment:
/etc/sonic/graph_service_url. A non-existing file means dhcp, and an exisiting file writing "disabled" means disabled.

---
In reply to: [100565302](https://github.com/Azure/sonic-buildimage/pull/288#discussion_r100565302) [](ancestors = 100565302)